YeoLab/clipper
# Move `data` outside of CLIPper such that users can easily make different reference w/o re-installing

to test:
download and re-install this branch.

`clipper -h` will see an option `--datadir=DATADIR     folder that stores *AS.STRUCTURE.gff`

notice in setup.py package_data is commented out s.t. data/ is no longer installed

try with the native data directory
`clipper -b clipper/test/data/allup_test.bam -o test.clipper.out --datadir=clipper/data/ --species=hg19 --gene=ENSG00000198901`

now move data to any arbitrary location
```
mkdir ~/scratch/clipper_data_random_place
cp clipper/data/hg19* ~/scratch/clipper_data_random_place
mkdir ~/scratch/clipper_data_random_place/regions
cp clipper/data/regions/hg19* ~/scratch/clipper_data_random_place/regions
```

re-run:
`clipper -b clipper/test/data/allup_test.bam -o test.clipper.otherdatadir.out --datadir=$HOME/scratch/clipper_data_random_place/ --species=hg19 --gene=ENSG00000198901`
